backend/api/serializers/notification_serializer.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+from api.models import Notification
+
+class NotificationSerializer(serializers.ModelSerializer):
```

Comment:
This folder backend/api/ doesn't exist - so this serializer is redundant and can be removed